src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -387,6 +393,13 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'get_policy_setdefinition')
         g.custom_command('update', 'update_policy_setdefinition')
 
+    with self.command_group('policy exemption', resource_policy_exemptions_sdk, resource_type=ResourceType.MGMT_RESOURCE_POLICY, min_api='2020-09-01') as g:
+        g.custom_command('create', 'create_policy_exemption')
+        g.custom_command('delete', 'delete_policy_exemption')
+        g.custom_command('list', 'list_policy_exemption')
+        g.custom_show_command('show', 'get_policy_exemption')
+        g.custom_command('update', 'update_policy_exemption')
```

Comment:
could we use generic update command for you update? https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#generic-update-commands

Reply:
As other policy resource, the policy exemption doesn't support patch. And using generic kind of break the consistency.

Reply:
then what is the difference between create and update? Generic update command is a CLI solution for operations without PATCH to provide a PATCH-like behavior.

Reply:
I'd keep the consistency with other policy commands.